pom.xml

```diff
@@ -963,4 +971,37 @@
     </snapshotRepository>
   </distributionManagement>
 
+  <profiles>
+    <!-- Spark version profile -->
+    <profile>
+      <id>spark-1.6</id>
+      <activation>
+        <property>
+          <name>!spark-2.0</name>
+        </property>
+      </activation>
+      <properties>
+        <spark.version>1.6.2</spark.version>
+        <py4j.version>0.9</py4j.version>
+        <netty.version>4.0.23.Final</netty.version>
+        <json4s.version>3.2.10</json4s.version>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>spark-2.0</id>
```

Comment:
Because Spark 2.0 upgraded some dependencies like Netty, json4s-jackson which are not compatible with Spark 1.6.

1. Currently we will upload netty jar as livy-rsc dependencies to Spark, and this will introduce conflict since we use the old version.
2. json4s-jackson's `render` API signature is changed in Spark's version.

That's why I use separated profiles for Spark 1.6 and 2.0. Yes this can also be avoided either with shading or some other ways. 

@vanzin what is your suggestion? Thanks a lot.